src/script.js

```diff
@@ -161,6 +176,7 @@ function isCanonicalSignature (buffer) {
 module.exports = {
   compile: compile,
   decompile: decompile,
+  toWitness: toWitness,
```

Comment:
It could be called evalPushOnly (there's a complementary function PushAll from Core which does the reverse: 
https://github.com/bitcoin/bitcoin/blob/2f71490d21796594ca6f55e375558944de9db5a0/src/script/sign.cpp#L124)

It feels like it belongs in script. I plan on passing scriptSig's through this to prepare for signature extraction, because then extraction can work against an eval'd scriptSig, or a witness as is, making the function mostly agnostic to segwit.